src/components/TodosList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+function TodosList(props) {
+  const { todos } = props;
+  const { handleChangeProps } = props;
+  const { deleteTodoProps } = props;
```

Comment:
Same suggestion here for object destructuring

Reply:
Done